airflow/www/views.py

```diff
@@ -5146,8 +5147,13 @@ def varimport(self):
             flash("Missing file or syntax error.", "error")
             return redirect(self.get_redirect())
         else:
+            skipped = set()
             suc_count = fail_count = 0
             for k, v in variable_dict.items():
+                if session.scalar(select(models.Variable).where(models.Variable.key == k)):
```

Comment:
This can use `exists_query` instead

Reply:
Seems like this will be better since we are not querying a lot of values